__tests__/linkdrop.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { Runner, toYocto, createKeyPair, BN, tGas } from "..";
+
+describe(`Running on ${Runner.getNetworkFromEnv()}`, () => {
+  jest.setTimeout(60000);
+  let runner: Runner;
+
+  beforeAll(async () => {
+    runner = await Runner.create(async ({ runtime }) => ({
+      linkdrop: await runtime.createAndDeploy(
+        "linkdrop",
+        `${__dirname}/build/debug/linkdrop.wasm`
+      ),
+    }));
+  });
+
+  test("call myself", async () => {
+    await runner.run(async ({ root, linkdrop }) => {
+      // Create temporary keys for access key on linkdrop
+      const senderKey = createKeyPair();
+      const public_key = senderKey.getPublicKey().toString();
+
+      // This adds the key as a function access key on `create_account_and_claim`
+      await root.call(
+        linkdrop,
+        "send",
+        {
+          public_key,
+        },
+        {
+          attachedDeposit: toYocto("2"),
+        }
+      );
+      // can only create subaccounts
```

Comment:
can you add some detail to this comment? the code below is still pretty hard to understand, even with this

Reply:
actually, I think the code might be more readable without this particular comment

```suggestion
```